apiary.apib

```diff
@@ -1142,7 +1142,13 @@ Get details for a single invoice.
                         + rate: 0.21 (number)
                         + taxable (Money)
                         + tax (Money)
+                + due: 0 (number)
```

Comment:
Q : doesn't `due` need to be added to the `.list` call too?

Reply:
I guess, but it isn't there yet